cli/gcp.go

```diff
@@ -136,6 +136,10 @@ func newGCPGKECmd() cli.Command {
 				Name:  "name, n",
 				Usage: "Name of the cluter",
 			},
+			cli.StringFlag{
+				Name: "namespace, ns",
+				Usage: "Specify the namespaces of workload page to open (you can input multiple namespaces by comma speparated string)",
```

Comment:
Typo ✏️  I also want to remove the `Specify xxx` because I want to unify with the existing flags. 

```suggestion
				Usage: "Namespaces of workload page to open (you can input multiple namespaces by comma-separated string)",
```